.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: |
           mkdir build
           cd build
-          cmake -DOPENSSL_ROOT_DIR=/usr/local/Cellar/openssl@1.1/1.1.1i -DCMAKE_INSTALL_PREFIX=/usr/local -DBUILD_TESTING=ON -DBUILD_TESTING_INTEGRATION=ON -DC_WARNINGS_AS_ERRORS=ON -DCPP_WARNINGS_AS_ERRORS=ON ..
+          cmake -DOPENSSL_ROOT_DIR="$(ls -td -- /usr/local/Cellar/openssl@1.1/* | head -n 1)" -DCMAKE_INSTALL_PREFIX=/usr/local -DBUILD_TESTING=ON -DBUILD_TESTING_INTEGRATION=ON -DC_WARNINGS_AS_ERRORS=ON -DCPP_WARNINGS_AS_ERRORS=ON ..
```

Comment:
Depending on the directory names and timestamps the `ls -rd` might be a better choice instead of `ls -td`.

If the directory names are based on the openssl version and the time values of the directories is the installation date, then the `ls -rd` results in the latest openssl version, even if an older one was installed after the latest one. This is quite uncommon I think, but if we can cover this case easily, then we should.

Reply:
I agree! The update is pushed :smile: